sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/src/Customization/MediaGraphPemCertificateList.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace Azure.Media.VideoAnalyzer.Edge.Models
+{
+    /// <summary>
+    /// Initializes a new instance of the MediaGraphPemCertificateList class.
+    /// </summary>
+    public partial class PemCertificateList
```

Comment:
Since this does not implement `IList<T>`, `IRdadOnlyList<T>`, etc., it should not be called "List". We normally use plural forms here, like `PemCertificates`. Name should be reflected across languages.

Reply:
Unfournately, the generated model is called "PemCertificateList" and this is a partial class of that so I don't think we can rename unless we change the swagger